cloud/pkg/dynamiccontroller/application/eventhandler.go

```diff
@@ -87,10 +86,10 @@ type CommonResourceEventHandler struct {
 	listenersLock sync.RWMutex
 	messageLayer  messagelayer.MessageLayer
 	gvr           schema.GroupVersionResource
-	informer      informers.GenericInformer
+	informer      *genericinformers.InformerPair
```

Comment:
InformerPairs is only used here, can be refactored

Reply:
I don't quite understand what you mean, can you elaborate?

Reply:
updated, PTAL